tests/h/search/query_test.py

```diff
@@ -1,4 +1,3 @@
-# pylint: disable=too-many-lines
```

Comment:
The refactors here took this file under 1000 lines. Still some work to do in the brevity department.